util/log/file.go

```diff
@@ -38,9 +39,41 @@ import (
 // MaxSize is the maximum size of a log file in bytes.
 var MaxSize uint64 = 1024 * 1024 * 10
 
-// If non-empty, overrides the choice of directory in which to write logs.
-// See createLogDirs for the full list of possible destinations.
-var logDir *string
```

Comment:
Switch `logDir` from a pointer to a value was unrelated to the rest of this change. Don't beat me up over it. The reason it was a pointer was a hold over from when this was initialized by the glog package as `var logDir = flag.String(...)`.